webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelectActions.js

```diff
@@ -0,0 +1,31 @@
+import {
+  INIT,
+  UPDATE_OPTIONS,
+  UPDATE_SELECTED,
+} from './TypeAheadSelectConstants';
+import { mapSelected } from './TypeAheadSelectSelectors';
+
+export const initialUpdate = (options, selected, id) => ({
```

Comment:
Perhaps `addTypeAheadSelect / TYPEAHEAD_ADD` would be a better name for this action?  `initialUpdate` doesn't make clear _why_ it needs to be initially updated.

Reply:
How about createTypeAheadSelect / TYPEAHEAD_CREATE ?

Reply:
I gave this a re-think. I think 'init' actually makes sense because it's initializing the redux store for the component and it's also consistent in name with the AutoComplete component. I suggest leaving the name if that's alright